src/components/GridView.tsx

```diff
@@ -0,0 +1,47 @@
+import { Col, Icon, Row } from '@antmjs/vantui';
+import { View } from '@tarojs/components';
+import { FC } from 'react';
+
+import { descriptionItems, homeItems } from '../api/home';
+
+export const HomeGridLayout: FC = () => (
+  <View>
+    <DescriptionView items={descriptionItems} />
+    <GridView items={homeItems} />
+  </View>
+);
+
+export type IconText = Record<'icon' | 'text', string>;
+
+export type Icons = Partial<Record<'items', IconText[]>>;
+
+export type DescriptionViewProps = Icons;
+
+export const DescriptionView: FC<DescriptionViewProps> = ({ items }) => (
+  <Row className="text-center my-2">
+    {items?.map(({ icon, text }, index) => (
+      <Col span={8} key={index} className="text-nowrap">
+        <Icon name={icon} className="idea-text-yellow me-1" />
+        {text}
+      </Col>
+    ))}
+  </Row>
+);
+
+export type GridViewProps = Icons;
+
+// todo: rename this variable
+export const GridView: FC<GridViewProps> = ({ items }) => {
+  return (
+    <Row>
+      {items?.map(({ icon, text }, index) => (
+        <Col className="text-center" key={index} span={6}>
```

Comment:
```suggestion
export const GridView: FC<GridViewProps> = ({ items }) => (
    <Row>
      {items?.map(({ icon, text }) => (
        <Col className="text-center" key={text} span={6}>
```
`key` 尽量不用数组索引，而用**顺序无关的唯一值**